hathor/websocket/factory.py

```diff
@@ -197,7 +197,10 @@ def serialize_message_data(self, event, args):
             data = tx.to_json_extended()
```

Comment:
Why hasn't the mempool API unittest gotten it in the first place? Any issue in the simulator?

Reply:
I don't think the simulator should have caught this problem as it is fairly independent from the main flow. But regardless, I don't remember seeing this exception on the simulator logs (which I've been running a lot lately). Maybe it's setup in a way that doesn't run that part of the code.

Reply:
Also this isn't a feature of the mempool API itself. It's a feature that was included together with the mempool API because the wallet-service mempool-sync needed it.